### ISSUE TITLE
Move into Fedora?

### ISSUE THREAD

**Issue by mavit** [fedberry/wiringpi#2]
Do you think it would be reasonable to move this package into Fedora proper, even though (as I understand it) it wouldn’t work there without the Raspberry Pi kernel?  I ask because then I could link against it from Fedora’s [Squeezelite](https://src.fedoraproject.org/rpms/squeezelite) builds for ARM, which seems like it might be less hassle than trying to get a special build of Squeezelite into FedBerry.  Is this a terrible idea?

**Comment by agrez** [comment]
Hi Peter. No, its not a terrible idea, I am all for pushing whatever we can upstream :-)

**Comment by mavit** [comment]
Great.

Are you a Fedora packager who can submit this for review?  I’d be happy to take the review in that case.

If not, would you like to become a Fedora packager?  Wiki page [Join the package collection maintainers](https://fedoraproject.org/wiki/Join_the_package_collection_maintainers?rd=PackageMaintainers/Join#How_to_join_the_Fedora_Package_Collection_Maintainers.3F) explains how to do that.

**Comment by agrez** [comment]
Nope, not a Fedora packager. Yes, I have been tempted to become a packager before and I suppose this is as good as time as any. I am not available for the next 1 1/2 days but when I get back I will read that mountain of info in that link and start the process.

**Comment by mavit** [comment]
The documentation can be a little bit maze-like, so feel free to ask me if you need any pointers.

**Comment by agrez** [comment]
Ok, review request done. Yep, that documentation has kept me from joining the Fedora packagers for years now. Its daunting to say the least!
https://bugzilla.redhat.com/show_bug.cgi?id=1577011

**Comment by pipiche38** [comment]
+1 to follow, as I would immediately jump and use it !
